pkg/virt-handler/virtwrap/schema_test.go

```diff
@@ -1,84 +1,36 @@
 package virtwrap
 
 import (
-	"encoding/json"
 	"encoding/xml"
 	"github.com/jeevatkm/go-model"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"kubevirt.io/kubevirt/pkg/api/v1"
 )
 
-var exampleXML = `
-<domain type="qemu">
+var exampleXML = `<domain type="qemu">
   <name>testvm</name>
   <memory unit="KiB">8192</memory>
   <os>
     <type>hvm</type>
   </os>
-    <devices>
-      <emulator>/usr/local/bin/qemu-x86_64</emulator>
-      <interface type='network'>
-        <source network='default'/>
-      </interface>
-      <disk type='network' device='disk'>
-        <driver name='qemu' type='raw'/>
-        <source protocol='iscsi' name='iqn.2013-07.com.example:iscsi-nopool/2'>
-          <host name='example.com' port='3260'/>
-        </source>
-      </disk>
-    </devices>
-</domain>
-`
-
-var exampleJSON = `
```

Comment:
Instead of deleting that, could you move the json test to pkg/api/v1/schema_tests.go and check if you can marshal and unmarshal a v1.DomainSpec?